framework/wazuh/rbac/orm.py

```diff
@@ -724,21 +737,23 @@ def add_user(self, username: str, password: str, check_default: bool = True):
         True if the user has been created successfully. False otherwise (i.e. already exists)
         """
         try:
-            user_id = None
             try:
                 if check_default and self.session.query(User).order_by(desc(User.id)
                                                                        ).limit(1).scalar().id < max_id_reserved:
                     user_id = max_id_reserved + 1
             except (TypeError, AttributeError):
                 pass
-            self.session.add(User(username=username, password=generate_password_hash(password), user_id=user_id))
+            self.session.add(User(username=username,
+                                  password=password if hash_password else generate_password_hash(password),
```

Comment:
I understand that, but yet:

```
        hash_password : bool
            Hash the password before saving it to the database if `True`. Save it raw otherwise.
```

This is not what the parameter is saying.

Reply:
We decided to change the parameter name and description.